src/perf/lib/ThroughputClient.cpp

```diff
@@ -377,13 +378,17 @@ ThroughputClient::StreamCallback(
         uint64_t ElapsedMicroseconds = StrmData->EndTime - StrmData->StartTime;
         uint32_t SendRate = (uint32_t)((StrmData->BytesCompleted * 1000 * 1000 * 8) / (1000 * ElapsedMicroseconds));
 
-        WriteOutput("[%p][%llu] Closed [%s] after %u.%u ms. (TX %llu bytes @ %u kbps).\n",
-            StrmData->Connection,
-            (unsigned long long)GetStreamID(MsQuic, StreamHandle),
-            "Complete",
-            (uint32_t)(ElapsedMicroseconds / 1000),
-            (uint32_t)(ElapsedMicroseconds % 1000),
-            (unsigned long long)StrmData->BytesCompleted, SendRate);
+        if (StrmData->BytesCompleted != 0) {
```

Comment:
It's an acceptable scenario to open and close a stream with no data. Not necessarily a failure.

Reply:
Even for throughput? I'd consider that an error. If not, I'll need to find another way to detect failure. Especially since I wouldn't be able to use Shutdown initiated by transport.